cypress/integration/default/learn/header/universal-navigation.js

```diff
@@ -64,6 +65,24 @@ describe('Language menu', () => {
   });
 });
 
+describe('Language menu with keyboard', () => {
+  before(() => {
+    cy.get(selectors['exit-lang-menu']).click();
+    cy.get(selectors['navigation-list']).contains('Change Language').blur();
+  });
+
+  it('should show language options.', () => {
+    cy.get(selectors['navigation-list'])
+      .contains('Change Language')
+      .type('{enter}');
+    cy.get(selectors['language-menu']).should('be.visible');
+  });
```

Comment:
Test for:
> checking that the language selector opens with enter

CC @moT01

Reply:
Hi @saw-jan, unfortunately this isn't enough, as what should be tested is 10 things as mentioned by bbsmooth [comment](https://github.com/freeCodeCamp/freeCodeCamp/pull/46644#issuecomment-1210022946)

You can use [jQuery to test for tab buttons](https://docs.cypress.io/api/commands/window#Notes), this maybe a taxing task, so I am changing this to ref instead of close the issue

Reply:
I will shortly add the tests listed in [comment](https://github.com/freeCodeCamp/freeCodeCamp/pull/46644#issuecomment-1210022946)